Documentation/markdown/SupportedIFCrepresentations.md

```diff
@@ -24,7 +24,92 @@ Meaning of columns:
 | Unit test | does OIP have unit tests for this entity; denoted by symbol, see [legend above](#legend) |
 | Issue | Link to corresponding open issue(s) |
 
-| Entity                | Covered              | Converter                  | Unit test             | Issue |
-|-----------------------|----------------------|----------------------------|-----------------------|-------|
-| `IfcCartesianPoint`   | :heavy_check_mark:   | `PlacementConverter`       | :heavy_check_mark:    | -     |
-| `IfcPointOnCurve`     | :x:                  | -                          | -                     | [#359](https://www.github.com/tumcms/Open-Infra-Platform/issues/359) |
+| Entity                                | Covered              | Converter                  | Unit test             | Issue |
+|---------------------------------------|----------------------|----------------------------|-----------------------|-------|
+| `IfcAdvancedBrep`                     | :clock9:             | `SolidModelConverter`      | -                     | -     |
+| `IfcAdvancedBrepWithVoids`            | :clock9:             | `SolidModelConverter`      | -                     | -     |
```

Comment:
These should clearly throw an exception. Not supported.

Reply:
Resolved 31a5bd4